### ISSUE TITLE
Output from ovftool in vSphere post-processor is not shown in the log

### ISSUE THREAD

**Issue by andrewsav-datacom** [hashicorp/packer#5498]
Issue from here: https://groups.google.com/forum/#!topic/packer-tool/xIi0kkBEYeQ

When vSphere post-processor runs ovftool  stdout/stderr from ovftool gets swallowed. It would be better if it is posted to packer log, as there is no other way of getting it otherwise, and having it would help with debugging packer templates.

**Comment by mwhooker** [comment]
@andrewsav-datacom can you test the code in #5499?

**Comment by andrewsav-datacom** [comment]
@mwhooker sorry, not a golang programmer. If you can make the windows binary available, I can test. Thanks!

**Comment by mwhooker** [comment]
@andrewsav-datacom 
[5498.zip](https://github.com/hashicorp/packer/files/1408985/5498.zip) amd_64

**Comment by andrewsav-datacom** [comment]
@mwhooker yep, can see the output now, both for success and for error. It looks like it displays the output twice, in case of error, the second time marked with "telemetry". That's probably fine, just letting you know in case if it is not. Thank you for the quick fix!

**Comment by mwhooker** [comment]
just with `PACKER_LOG=1` turned on, right? can you paste the output when it's an error?

**Comment by andrewsav-datacom** [comment]
>just with PACKER_LOG=1 turned on, right?
yes

Here is the relevant part:

```log
2017/10/24 13:49:43 ui: [1;32m==> vmware-iso: Running post-processor: vsphere[0m
2017/10/24 13:49:43 [INFO] (telemetry) Starting post-processor vsphere
2017/10/24 13:49:43 ui: [0;32m    vmware-iso (vsphere): Uploading vm-template-rancheros\rancheros.vmx to vSphere[0m
2017/10/24 13:49:43 packer.exe: 2017/10/24 13:49:43 Starting ovftool with parameters: --acceptAllEulas --name=template-rancheros-test --datastore=<datastore-redacted> --noSSLVerify=true --diskMode=thick --vmFolder=/WIP --network=Admin Network --overwrite --annotation built with packer vm-template-rancheros\rancheros.vmx vi://<domain-redacted>%5C<account-redacted>:<password>@<vCenter-redacted>/<host-redacted>/host/<cluster-redacted>
2017/10/24 13:49:43 [INFO] (telemetry) ending vsphere
2017/10/24 13:49:43 [INFO] (telemetry) found error: Failed: exit status 1
Error: Unexpected option: vi://<domain-redacted>%5C<account-redacted>:<password-redacted>@<vCenter-redacted>/<host-redacted>/host/<cluster-redacted>
Completed with errors

2017/10/24 13:49:43 Deleting original artifact for build 'vmware-iso'
2017/10/24 13:49:43 ui error: [1;31mBuild 'vmware-iso' errored: 1 error(s) occurred:

* Post-processor failed: Failed: exit status 1
Error: Unexpected option: vi://<domain-redacted>%5C<account-redacted>:<password-redacted>@<vCenter-redacted>/<host-redacted>/host/<cluster-redacted>
Completed with errors

[0m
2017/10/24 13:49:43 Builds completed. Waiting on interrupt barrier...
2017/10/24 13:49:43 machine readable: error-count []string{"1"}
2017/10/24 13:49:43 ui error: 
==> Some builds didn't complete successfully and had errors:
2017/10/24 13:49:43 machine readable: vmware-iso,error []string{"1 error(s) occurred:\n\n* Post-processor failed: Failed: exit status 1\nError: Unexpected option: vi://<domain-redacted>%5C<account-redacted>:<password-redacted>@<vCenter-redacted>/<host-redacted>/host/<cluster-redacted>\r\nCompleted with errors\r\n\n"}
2017/10/24 13:49:43 ui error: --> vmware-iso: 1 error(s) occurred:

* Post-processor failed: Failed: exit status 1
Error: Unexpected option: vi://<domain-redacted>%5C<account-redacted>:<password-redacted>@<vCenter-redacted>/<host-redacted>/host/<cluster-redacted>
Completed with errors

2017/10/24 13:49:43 ui: 
==> Builds finished but no artifacts were created.
2017/10/24 13:49:43 [INFO] (telemetry) Finalizing.
2017/10/24 13:49:44 waiting for all plugin processes to complete...
2017/10/24 13:49:44 E:\pkg\windows_amd64\packer.exe: plugin process exited
2017/10/24 13:49:44 E:\pkg\windows_amd64\packer.exe: plugin process exited
2017/10/24 13:49:44 E:\pkg\windows_amd64\packer.exe: plugin process exited
2017/10/24 13:49:44 E:\pkg\windows_amd64\packer.exe: plugin process exited
```
On a side note the password is displayed here unmasked in the log. It probably needs to be substituted. The same way it's substituted in the `Starting ovftool with parameters` line.

**Comment by mwhooker** [comment]
that's great, thanks!

**Comment by andrewsav-datacom** [comment]
@mwhooker can you comment on password substitution please?

**Comment by mwhooker** [comment]
fixed in 63aa784df

**Comment by AndrewSav** [comment]
@mwhooker Thanks!
